libs/clients/regulations/src/lib/regulations.ts

```diff
@@ -26,9 +26,9 @@ export const REGULATIONS_OPTIONS = 'REGULATIONS_OPTIONS'
 
 export interface RegulationsServiceOptions {
   url: string
-  ttl?: number
 }
 
+@Injectable(/*{ scope: Scope.REQUEST }*/)
```

Comment:
Clean this up?

Reply:
...ef ég skil ykkur Val rétt: https://github.com/island-is/island.is/pull/5865/commits/a025c88fca138930858ee56fd110eba95fad3a34